packages/contracts-ops/contracts/scripts/DeployImplementations.s.sol

```diff
@@ -64,92 +64,80 @@ abstract contract DeployImplementationsLogic is Script, Config {
 }
 
 contract DeployImplementations is DeployImplementationsLogic {
-    using JsonWriter for JsonWriter.Buffer;
-    using JsonWriter for string;
-
-    JsonWriter.File outputFile;
-
     // entrypoint
-    function deploy(
-        string memory _configPath,
-        string memory _outputFile,
-        string memory _domain
-    ) public {
+    function deploy(string memory _configPath, string memory _domain) public {
         __Config_initialize(_configPath);
-        // begin output file
-        _outputFile = string(abi.encodePacked("./actions/", _outputFile));
-        outputFile.path = _outputFile;
-        outputFile.overwrite = false;
-        JsonWriter.Buffer memory buffer = JsonWriter.newBuffer();
-        string memory indent = "";
-        buffer.writeObjectOpen(indent);
-        // loop through domains and deploy implementations
         vm.createSelectFork(getRpcs(_domain)[0]);
         vm.startBroadcast();
         deployImplementations(_domain);
-        write(buffer, indent.nextIndent(), _domain);
-        // write final output
-        buffer.writeObjectClose(indent, true);
-        buffer.flushTo(outputFile);
+        updateImpl(_domain, _configPath);
     }
 
-    function write(
-        JsonWriter.Buffer memory buffer,
-        string memory indent,
-        string memory _domain
-    ) private {
-        // start
-        buffer.writeObjectOpen(indent, _domain);
-        string memory inner = indent.nextIndent();
-        // write implementations
-        writeImplementation(buffer, inner, "home", address(home), false);
-        writeImplementation(buffer, inner, "replica", address(replica), false);
-        writeImplementation(
-            buffer,
-            inner,
-            "governanceRouter",
-            address(governanceRouter),
-            false
+    function updateImpl(string memory _domain, string memory _configPath)
+        internal
+    {
+        string memory valueKey = string(
```

Comment:
this will read a bit nicer if you use the utils in the `Config.sol` for generating paths in the config :)

Reply:
then you can just pass that directly in vm.writeJson as a param, instead of overwriting the same memory variable repeatedly